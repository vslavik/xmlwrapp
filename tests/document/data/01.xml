<?xml version="1.0"?>
<?xslt stylesheet="foo"?>
<!--comment-->
<root/>
<!--comment-->
