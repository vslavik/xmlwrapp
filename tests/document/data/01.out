type_pi
type_comment
type_element
type_comment
